lib/std/os.zig

```
@@ -3745,7 +3746,7 @@ pub fn sendfile(
                     EPIPE => return error.BrokenPipe,
 
                     else => {
-                        _ = unexpectedErrno(err);
+                        const discard = unexpectedErrno(err);
```

Why does this need an assignment?

---

because it's an error

---

then should it be: `unexpectedErrno(err) catch {}`?

---

hrm. why don't we support discarding error values (not error *unions*) via assignment to `_`?